backend/controllers/addAppointmentDetails.js

```diff
@@ -2,7 +2,7 @@ const moment = require('moment')
 const config = require('../config')
 const { serviceUnavailableMessage } = require('../common-messages')
 const { DATE_TIME_FORMAT_SPEC, DAY_MONTH_YEAR, buildDateTime } = require('../../src/dateHelpers')
-
+const { properCaseName } = require('../utils')
```

Comment:
Not a show stopper but wondering if this is better off being called `capitalise` now it's being used for things other than names.

Reply:
That's a good spot!

Reply:
Will change it on the next PR

Reply:
Oh, there is already a `capitalize` in utils